Src/Base/AMReX_FabArrayBase.cpp

```diff
@@ -661,7 +663,9 @@ FabArrayBase::FB::define_fb(const FabArrayBase& fa)
     
     const int nlocal = imap.size();
     const IntVect& ng = m_ngrow;
+    const IntVect ng_ng(AMREX_D_DECL(m_ngrow[0]-1,m_ngrow[1]-1,m_ngrow[1]-1));
```

Comment:
There is an index bug here.  The last one should be `[2]`.  You could also do `const IntVect ng_ng = m_ngrow - 1;`.

Reply:
Done